tools/topology/CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ set(TPLGS
 	"sof-tgl-rt1308-nohdmi\;sof-tgl-rt1308-nohdmi"
 	"sof-imx8qxp-nocodec\;sof-imx8qxp-nocodec"
 	"sof-imx8qxp-cs42888\;sof-imx8qxp-cs42888"
+	"sof-cml-da7219-max98357a\;sof-cml-da7219-max98357a"
```

Comment:
maybe a dumb question, but this machine driver is shared with glk_da7219_max98357a, so can't we share the same topology file and use variables to adjust SSP values?

Reply:
This commit i have cherry picked from cml-007-drop-stable branch. (05c8e96).
This topology file is the one which is used in the SOF 007 release.
@macchian could you please comment on this ?.

Reply:
Per request, reuse the glk topology and update new PR https://github.com/thesofproject/sof/pull/2397 for reviewing.